lab-teddy/test/collecting-data-test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Buffer = require('buffer').Buffer;
+const fs = require('fs');
+const dataFiles = require('../lib/collecting-data.js');
+let mocData = [ '436f707972696768', '4c6f72656d206970', '4c6f72656d206970' ];
+console.log(exports);
+describe('collecting-data.js', function(){
+  it('should be an object', done => {
+    fs.readFile(`${__dirname}/../data/one.text`, function(err, data){
```

Comment:
So you are not actually testing this correctly. Instead of using the fs module. You actually need to test your lib function. So this would look more like dataFiles('filepath', callback() {}).